src/main.cc

```diff
@@ -476,11 +476,14 @@ load_cfg(void)
 	 * logger init must happen before daemonising in order for the error
 	 * to show and for the process to exit with a failure status
 	 */
-	say_logger_init(log,
-			cfg_geti("log_level"),
-			cfg_getb("log_nonblock"),
-			log_format,
-			background);
+	if (luaT_dostring(tarantool_L, "require('log').box_api.cfg()") != 0) {
+		diag_log();
+		panic("failed to configure log");
```

Comment:
Nit: Why `panic` here? Why not `exit` as in other places? Looks inconsistent.

Reply:
Fixed.